--- include/Make/Platform.make.in.orig	2020-09-18 20:22:59.000000000 +0200
+++ include/Make/Platform.make.in	2020-09-19 09:21:53.000000000 +0200
@@ -37,7 +37,7 @@
 exec_prefix         = @exec_prefix@
 ARCH                = @host@
 UNIX_BIN            = @BINDIR@
-INST_DIR            = @INSTDIR@
+INST_DIR            = @exec_prefix@
 
 GRASS_HOME          = @GRASS_HOME@
 RUN_GISBASE         = @GISBASE@
@@ -62,7 +62,7 @@
 
 #shared libs
 SHLIB_PREFIX        = lib
-SHLIB_LD            = @SHLIB_LD@
+SHLIB_LD            = @SHLIB_LD@ ${ICONVLIB}
 SHLIB_LDFLAGS       = @SHLIB_LD_FLAGS@
 SHLIB_CFLAGS        = @SHLIB_CFLAGS@
 SHLIB_SUFFIX        = @SHLIB_SUFFIX@
@@ -212,7 +212,7 @@
 CAIRO_HAS_XRENDER_SURFACE = @CAIRO_HAS_XRENDER_SURFACE@
 
 #Python
-PYTHON              = python3
+PYTHON              = ${prefix}/bin/python
 
 #regex
 REGEXINCPATH        = @REGEXINCPATH@
