--- utils/mkhtml.py.orig	2022-01-29 17:26:38.000000000 +0100
+++ utils/mkhtml.py	2022-01-29 17:32:19.000000000 +0100
@@ -48,7 +48,13 @@
     import grass.script as gs
 except ImportError:
     # During compilation GRASS GIS
-    gs = None
+    _ = str
+    class gs:
+        def warning(message):
+            pass
+
+        def fatal(message):
+            pass
 
 HEADERS = {
     "User-Agent": "Mozilla/5.0",
